src/bramble/client/StateManager.js

```diff
@@ -103,8 +116,8 @@ define(function() {
                 set: function(v) { storage.setItem(prefix("wordWrap"), v); }
             },
             autoCloseTags: {
-                get: function()  { return getBool(storage, "closeTags"); },
-                set: function(v) { storage.setItem(prefix("closeTags"), v); }
+                get: function()  { return getObject(storage, "closeTags"); },
+                set: function(v) { storage.setItem(prefix("closeTags"), JSON.stringify(v)); }
```

Comment:
this may need a `setObject` function wrapper, as JSON.stringify can throw when you try to serialize objects. If there are cyclic references `stringify` cannot complete its task and will throw a TypeError, and if there are functions, stringify will strip them.